Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -626,19 +586,21 @@ AppDelegateViewModelOutputs {
 
     // Koala
 
-    self.notificationAuthorizationStatusProperty.signal
-      .skipNil()
-      .skip(until: self.notificationAuthorizationCompletedProperty.signal.ignoreValues())
-      .take(first: 1)
-      .observeValues { status in
-        switch status {
-        case .authorized:
-          AppEnvironment.current.koala.trackPushPermissionOptIn()
-        case .denied:
-          AppEnvironment.current.koala.trackPushPermissionOptOut()
-        case .notDetermined, .provisional: ()
-        }
+    Signal.combineLatest(
+      pushTokenRegistrationStartedValues,
+      pushNotificationsPreviouslyAuthorized
+    )
+    .filter { _, previouslyAuthorized in !previouslyAuthorized }
+    .map { isGranted, _ in isGranted }
+    .skipNil()
+    .take(first: 1)
+    .observeValues { isGranted in
+      if isGranted {
+        AppEnvironment.current.koala.trackPushPermissionOptIn()
+      } else {
+        AppEnvironment.current.koala.trackPushPermissionOptOut()
```

Comment:
These Koala events should track the same as before, no need to do the `@available` check anymore as that's handled in `PushRegistration`.